modules/ossm-performing-inplace-update.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: Procedure
+[id="performing-inplace-updtate"]
+= Performing InPlace update
+:context: ossm-performing-inplace-update
+
+When updating {istio}, you can only increment by one minor version at a time. If you are updating by more than one minor version, then you must increment the version and restart after each update. This ensures that the sidecar version is compatible with the control plane version. After all the workloads are restarted the update process is complete.
```

Comment:
The first sentence should mention that it only applies when the strategy is `InPlace`.

Reply:
done